modules/acs-architecture-connections.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * architecture/acs-architecture.adoc
+:_mod-docs-content-type: CONCEPT
+[id="acs-architecture-connections_{context}"]
+= {product-title-short} connection protocols and default ports
+
+The protocols and default ports for connections within {product-title-short} and between {product-title-short} and external components are shown in the following table.
+
+.{product-title-short} connections between components
+
+[cols="3",options="header"]
+|===
+| Component or external entity | Connection type | Port
+
+| Central and external services, for example, Jira and PagerDuty
+| HTTPS
+| ?
+
+| Central and secured cluster
```

Comment:
The primary communication between Central and the Cluster happens through Sensor.
Sensor could be seen as the proxy for everything that is running in a Secured Cluster.
I don't know whether we should split out Central <-> Secured Cluster comms in here.